buildSrc/subprojects/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/UpdateBranchStatus.kt

```diff
@@ -40,7 +40,7 @@ abstract class UpdateBranchStatus : DefaultTask() {
     private
     fun publishBranchStatus(branch: String) {
         println("Publishing branch status of $branch")
-        project.execAndGetStdout("git", "push", "origin", "$branch:green-$branch")
+        project.execAndGetStdout("git", "push", "https://bot-teamcity:${project.property("githubToken")}@github.com/gradle/gradle.git", "$branch:green-$branch")
```

Comment:
I wonder if there is a better way to pass the github token instead of using the URL. Do we log the command somewhere exposing the URL?

Reply:
No we don't log it. This is no different than other usage of tokens. The upside of this method is that we can easily update the token. WDYT @JLLeitschuh ?